cluster/addons.go

```diff
@@ -231,7 +231,7 @@ func (c *Cluster) deployKubeDNS(ctx context.Context) error {
 	log.Infof(ctx, "[addons] Setting up %s", c.DNS.Provider)
 	KubeDNSConfig := KubeDNSOptions{
 		KubeDNSImage:           c.SystemImages.KubeDNS,
-		KubeDNSSidecarImage:    c.SystemImages.KubeDNSSidecar,
+		KubeDNSSidecarImage:    DefaultRKETools,
```

Comment:
This actually `gcr.io/google_containers/k8s-dns-sidecar-amd64`, not `rke-tools`.

Reply:
thank you, mistook it for Kubernetes sidecar